backend/run.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+if [[ -n "${ENV}" ]]; then
+  echo "Environment set as: ${ENV}"
+else
+  echo "No environment variable `ENV` is set!"
+fi;
+
+export AWS_ACCESS_KEY_ID=test
+export AWS_SECRET_ACCESS_KEY=test
+
+#if [[ "${ENV}" == "LOCAL" ]] || [[ "${ENV}" == "TESTING" ]]; then
```

Comment:
hangover debugging code - need to put this conditional back in 😁

Reply:
Oops... fixed!